kalite/distributed/backup.py

```diff
@@ -0,0 +1,36 @@
+import dbbackup
+import datetime
+import os
+import sys
+import time
+from os.path import expanduser
```

Comment:
@benjaoming : Yes, you're right. Multiple os methods are being used, so importing "os" once is sufficient. Thanks! I'll fix that.